src/ansys/turbogrid/core/multi_blade_row/summary_template.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <title>TurboGrid Multi Blade Row Summary</title>
+  <style>
+    table,
+    th,
+    td {
+      border: 1px solid
+    }
+
+    tr:first-child td,
+    tr td:first-child {
+      font-weight: bold;
+    }
+
+    td {
+      width: "15%";
+      text-align: center;
+    }
+
+    tr td:first-child {
+      width: "40%";
+      text-align: left;
+    }
+  </style>
+</head>
+
+<body>
+
+  <h1>TurboGrid Multi Blade Row Summary</h1>
+  
+  <p>Case: {{case_name}}</p>
+
```

Comment:
The headings should be consistent about whether they use capital letters for all words or just the first. Maybe use "Mesh Size Summary" and (lower down) "Meshing Errors".